src/sass/_login.scss

```diff
@@ -0,0 +1,9 @@
+.login {
+  display: flex;
```

Comment:
I don't think this `display: flex` is necessary, `.login` is applied to a single `div`